src/DataTable/DropdownFilters.jsx

```diff
@@ -47,6 +53,8 @@ DropdownFilters.propTypes = {
     /** React function component to display the filter */
     Filter: requiredWhen(PropTypes.func, 'canFilter'),
   })).isRequired,
+  /** Number between one and four filters that can be shown on the top row. */
+  numBreakoutFilters: PropTypes.oneOf([1, 2, 3, 4]),
```

Comment:
Is 0 valid?

Reply:
I guess? Zero will be chosen for you if your screen is too small.